### PR TITLE
initial updates for showing all groups for monitor based SLOs

### PR DESCRIPTION
**Pull request by roxanne-moslehi**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by roxanne-moslehi** [comment]
@estherk15 this is ready to review but we need to hold off on merging since it's a new feature to release!

**Review by estherk15 — APPROVED** [review]

File: content/en/service_management/service_level_objectives/guide/slo_types_comparison.md
Comment (roxanne-moslehi) [reply content/en/service_management/service_level_objectives/guide/slo_types_comparison.md:31]: @estherk15 So if your SLO has 5000 groups or less, you can see all the groups. If it has more than 5,000 groups you can't see any (technical limitations). Lmk if you think there is a better wording option

Comment (estherk15) [reply content/en/service_management/service_level_objectives/guide/slo_types_comparison.md:31]: Is this common? Do a lot of customer have SLOs with more than 5,000 groups? If not, recommend adding this limitation to the Support docs instead of public docs.
```suggestion
| **Functionality for SLO with Groups**                                               | SLO calculated based on all groups<br><br>Can view all groups in SLO side panel and SLO summary widget                                                                                                                  | Supported for SLOs with a single multi alert Monitor<br><br>**Option 1:** SLO calculated based on all groups (can view all groups in SLO side panel and SLO summary widget. Not supported for SLOs with more 5,000 groups)<br>**Option 2:** SLO calculated based on up to 20 selected groups (can view all selected groups in SLO side panel and SLO summary widget)                                                                                                                                | SLO calculated based on all groups<br><br>Can view all groups in SLO side panel and SLO summary widget                                                           |
```

**Comment by roxanne-moslehi** [comment]
@estherk15 I took out the note about 5,000 groups in the summary chart since that's more high level but left it in the more detailed pages. This is ready to merge when you approve it since the feature was released!

Also FYI @mmeyer724

**Pull request merged**
